### DISCUSSION TITLE
Offering torrent downloads

### DISCUSSION THREAD

**Discussion by EmanuelLoos** [Divested-Mobile/DivestOS-Build#63] [Ideas] [answered]
On the downloads page you write that server bandwidth is limited. I think offering torrent downloads could help a bit there. Additionally this would automate checking of the signature. Of course, there are probably many downloads for OTA updates as well so it might be a good idea to implement this in the updater as well (user controlled and not seeding using mobile data), if possible.
How about that?

**Comment by SkewedZeppelin** [comment, marked as answer]
This is not very effective. Especially on mobile.
Server bandwidth is not limited enough for me to consider such.
I'll probably add an EU mirror soon.

**Comment by SkewedZeppelin** [comment]
Not to mention the privacy issues...
